php/PhpJsonApiSamples/ClientSideEncryptionDemo1.html

```diff
@@ -0,0 +1,95 @@
+          in case of using different version of jQuery - refer this https://api.jquery.com/jquery.noconflict/-->
+        <!-- for the demo purpose, using CDN path -->
+        <script src="https://code.jquery.com/jquery-1.11.1.js"> </script>
+         
+        <script language="javascript">
+          
+                // set the key from an AJAX call  
+                $(document).ready(function(){
+                        paytrace.setKeyAjax('DemoPublic_Key.pem') ;// set the key from an AJAX call (in this case via a relative URL)
+                });
+                
+                $(document).ready( function(){
+                    $("#DemoForm").submit(function(e){
+                        //To prevent the default action of the submit
+                        e.preventDefault();
+                         
+                        // Do your validation and if all validations are met, 
+                        // Next is to submit the form with paytrace.submitEncrypted.
+                        if ($("#ccNumber").val() && $("#ccCSC").val()) {
+                           //if all validations are met, submit the form to Paytrace library for encyption.
+                            paytrace.submitEncrypted("#DemoForm");
+                        }
+                        else{ 
+                            alert("CCNumber and CSC number are required ! "); 
+                        }
+                    });
+                });  
+            
+              /*//you can use following code - different version of basic input validation
+              // set the key from an AJAX call  
+              $(document).ready(function(){
+                  paytrace.setKeyAjax('DemoPublic_Key.pem'); //<your.pem file> (may be using relative url)
+               });
+               
+              // 
+              $(document).ready( function(){
+                  $("#DemoForm").submit(function(e){
+                    // To prevent the Default action of the submit 
+                    e.preventDefault();
+                    
+                    // Do your validation and if all validations are met, 
+                    // Next is to submit the form with paytrace.submitEncrypted.
+                    var ccNumber = document.getElementById("ccNumber").value;
+                    var cscNumber = document.getElementById("ccCSC").value ;
+                    
+                    if(ccNumber.length == 0 ){
+                         alert("Please enter the CCnumber.");     
+                    }else if(cscNumber.length == 0){
+                          alert("Please enter the CSCnumber.");
+                    }else{
+                    //if all validation are met, submit the form to Paytrace library for encyption.
+                    paytrace.submitEncrypted("#DemoForm"); }
+                });    
+               });*/
+       
```

Comment:
Did you want to leave this commented out code here? Seems like it is demostrating the same feature as the above code and just written a little different. Seems like you can remove the commented out code and they can determine how they want to use the "submitEncrypted" feature.